internal/server/http/server.go

```diff
@@ -117,13 +119,20 @@ loop:
 func (s *Server) Run() {
 	go func() {
 		log.Printf("The fake server is on tap now: %s\n", s.httpServer.Addr)
-		err := s.httpServer.ListenAndServe()
+		err := s.run(s.tls)
 		if err != http.ErrServerClosed {
 			log.Fatal(err)
 		}
 	}()
 }
 
+func (s *Server) run(tls bool) error {
+	if tls {
+		return s.httpServer.ListenAndServeTLS("server.cert", "server.key")
```

Comment:
We have a problem with this solution, if you put the `server.cert` and `server.key` as file, when the user executes `killgrave` it will try to find those files on the same level, for sure that files will not exists in the user computer, so I propose a solution, create this `server.cert` and `server.key` as const on string and use:

```go
// Generate a key pair from your pem-encoded cert and key ([]byte).
cert, err := tls.X509KeyPair([]byte(serverCert), []byte(serverKey))

// Construct a tls.config
tlsConfig := &tls.Config{
  Certificates: []tls.Certificate{cert}
  // Other options
}

// Build a server:
server := http.Server{
    // Other options
    TLSConfig: tlsConfig,
}

// Finally: serve.
err = server.ListenAndServeTLS("", "")
```

Reply:
That's much nicer. Actually I did really bad committing cert files 🙈